pjp.tm08/src/main/java/com/subho/wipro/pjp/tm08/UsingPreparedStatementAndMetaDataObjects/Assignment2.java

```diff
@@ -0,0 +1,101 @@
+package com.subho.wipro.pjp.tm08.UsingPreparedStatementAndMetaDataObjects;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class Assignment2 {
+
+	public static void main(String[] args) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'main(String)' has an NPath complexity of 3600](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5628071)